net-libs/libutp/patches/libutp-20170101.patchset

```diff
@@ -0,0 +1,88 @@
+From 94aacb8daac300947f1acec4c700d1f3a47fd414 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Sun, 24 May 2020 21:30:14 +0200
+Subject: Haiku build fix
+
+
+diff --git a/utypes.h b/utypes.h
+index 79bafcd..5e786ee 100644
+--- a/utypes.h
++++ b/utypes.h
+@@ -1,5 +1,9 @@
+ #ifndef __UTYPES_H__
+-#define __UTYPES_H__
++#define __UTYPES_H__
++
++#ifdef __HAIKU__
++#include <SupportDefs.h>
++#endif
+ 
+ // standard types
+ typedef unsigned char byte;
+@@ -14,9 +18,11 @@ typedef signed int int32;
+ #ifdef _MSC_VER
+ typedef unsigned __int64 uint64;
+ typedef signed __int64 int64;
+-#else
++#else
++#ifndef __HAIKU__
+ typedef unsigned long long uint64;
+-typedef long long int64;
++typedef long long int64;
++#endif
+ #endif
+ 
+ /* compile-time assert */
+-- 
+2.26.0
+
+
+From a297ce01d62c6cb454af87170a0dafa104bfab0f Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Sun, 24 May 2020 21:34:57 +0200
+Subject: Build shared lib
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b68a3f6..58a49c6 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -11,7 +11,7 @@ endif()
+ 
+ include_directories(.)
+ 
+-add_library(${PROJECT_NAME} STATIC
++add_library(${PROJECT_NAME} SHARED
```

Comment:
If they didn't intend this would be shipped as a shared library, maybe we shouldn't ship it as one. What do other distros do?